ubuntu2404_tensorrt/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM nvidia/tensorrt:24.12-py3
+LABEL description="Ubuntu 20.04 with dependencies to build TensorRT inference component"
+LABEL maintainer="Benjamin Huth <benjamin.huth@cern.ch"
+# increase whenever any of the RUN commands change
+LABEL version="1"
+
+# DEBIAN_FRONTEND ensures non-blocking operation (tzdata is a problem)
+ENV DEBIAN_FRONTEND noninteractive
+
+# install dependencies from the package manager.
+#
+# see also https://root.cern.ch/build-prerequisites
+RUN apt-get update -y \
+  && apt-get upgrade -y \
+  && apt-get install -y \
+    build-essential \
+    curl \
+    git \
+    freeglut3-dev \
+    libexpat-dev \
+    libftgl-dev \
+    libgl2ps-dev \
+    libglew-dev \
+    libgsl-dev \
+    liblz4-dev \
+    liblzma-dev \
+    libpcre3-dev \
+    libx11-dev \
+    libxext-dev \
+    libxft-dev \
+    libxpm-dev \
+    libxerces-c-dev \
+    libzstd-dev \
+    zstd \
+    ninja-build \
+    python3 \
+    python3-dev \
+    python3-pip \
+    rsync \
+    unzip \
+    zlib1g-dev \
+    ccache \
+    libsqlite3-dev \
+    time \
+  && apt-get clean -y
+
+# manual builds for hep-specific packages
+ENV GET curl --location --silent --create-dirs
+ENV UNPACK_TO_SRC tar -xz --strip-components=1 --directory src
+ENV PREFIX /usr/local
+ENV CUDA_ARCH 86
```

Comment:
Would it be worth adding CC 5.0, which is the oldest supported by CUDA 12? That guarantees that we can run on any supported GPU.

Reply:
actually this is a leftover from copying another file, I think this variable has no effect in that image.